### DISCUSSION TITLE
Avoid server logs from being automatically deleted

### DISCUSSION THREAD

**Discussion by gonzaiq** [NVIDIA/NVFlare#2179] [Q&A] [answered]
Hi all,

I am working with v.2.2.

After the training is stopped, aborted, or ended, the log file of the server for that particular job (e.g., 0a892c48-52d7-4313-abff-11f736d0a587/log.tx) is automatically deleted.

How could I avoid the server log from being deleted ? It could be useful for knowing why the job has stopped in the first place.

Thanks in advance.

**Comment by yhwen** [comment, marked as answer]
You can use the admin command "download_job JOB_ID" to download the full workspace data to your local workstation.

**Reply by gonzaiq** [reply]
Thank you for your answer!
